recipes/d4binding/build_htslib.sh

```
@@ -0,0 +1,63 @@
+
+cd htslib
+
+cat > config.h << CONFIG_H
+#define HAVE_LIBBZ2 1
+#define HAVE_DRAND48 1
+CONFIG_H
+
+perl -i -pe 's/hfile_libcurl\.o//g' Makefile
+	
+function is_musl() {
+	if [ ! -z $(echo $TARGET | grep musl) ]; then 
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)